chrome/content/zotero/tools/csledit.xul

```diff
@@ -26,241 +26,44 @@
-					params.setStringValue("state_data", "\t");
-					controller.doCommandWithParams(command, params);
-				}
-			}
-		}
-	]]>
-	</script>
+	<script src="csledit.js"/>
 	
 	<vbox flex="1">
 		<hbox align="center">
-		    <button id="preview-refresh-button" label="Refresh" oncommand="Zotero_CSL_Editor.refresh()"/>
-		    <button id="zotero-csl-save" label="Save" oncommand="Zotero_CSL_Editor.save()"/>
-		    <menulist id="zotero-csl-page-type" style="min-height: 1.6em; min-width: 50px" oncommand="Zotero_CSL_Editor.refresh()" />
-		    <label value=":" />
-		    <textbox size="5" id="preview-pages" type="timed" timeout="250" oncommand="Zotero_CSL_Editor.refresh()"/>
-		    <checkbox oncommand="Zotero_CSL_Editor.refresh()" id="preview-suppress-author" label="Suppress author" />
-		    <label value="Citation is:" />
-		    <menulist id="zotero-ref-position" oncommand="Zotero_CSL_Editor.refresh()">
-		    	<menupopup>
-		    	    <menuitem label="First" value="0"/>
-		    	    <menuitem label="Subsequent" value="1"/>
-		    	    <menuitem label="Ibid" value="2"/>
-		    	    <menuitem label="Ibid+Locator" value="3"/>
-		    	    <menuitem label="Near Note" value="4"/>
-		    	</menupopup>
-		    </menulist>
-		    <menulist id="zotero-csl-list" style="min-height: 1.6em; min-width: 100px" initialized="false" flex="1" oncommand="Zotero_CSL_Editor.loadCSL(this.selectedItem.value)"/>
+			<button id="preview-refresh-button" label="&zotero.csledit.refresh;" oncommand="Zotero_CSL_Editor.refresh()"/>
+			<button id="zotero-csl-save" label="&zotero.csledit.saveAs;" oncommand="Zotero_CSL_Editor.save()"/>
+			<menulist id="zotero-csl-page-type" style="min-height: 1.6em; min-width: 50px" oncommand="Zotero_CSL_Editor.refresh()" />
+			<label value=":" />
+			<textbox size="5" id="preview-pages" type="timed" timeout="250" oncommand="Zotero_CSL_Editor.refresh()"/>
+			<checkbox oncommand="Zotero_CSL_Editor.refresh()" id="preview-suppress-author" label="&zotero.csledit.suppressAuthor;" />
+			<label value="&zotero.csledit.citePosition;" />
+			<menulist id="zotero-ref-position" oncommand="Zotero_CSL_Editor.refresh()">
+				<menupopup>
+					<menuitem label="first" value="0"/>
+					<menuitem label="subsequent" value="1"/>
+					<menuitem label="ibid" value="2"/>
+					<menuitem label="ibid-with-locator" value="3"/>
+					<menuitem label="near-note" value="4"/>
+				</menupopup>
+			</menulist>
+			<menulist id="zotero-csl-list" style="min-height: 1.6em; min-width: 100px" initialized="false" flex="1" oncommand="Zotero_CSL_Editor.loadCSL(this.selectedItem.value)"/>
 		</hbox>
 		<textbox id="zotero-csl-editor" type="timed" timeout="250" multiline="true"
 			flex="1"
 			onkeypress="Zotero_CSL_Editor.handleKeyPress(event)"
 			oncommand="document.getElementById('zotero-csl-list').selectedIndex = -1; Zotero_CSL_Editor.generateBibliography(this.value)"/>
-		<splitter/>
+		<splitter id="csledit-splitter" collapse="before" persist="state">
```

Comment:
FYI, here's the relevant commit/comment from @simonster https://github.com/zotero/zotero/commit/47f57509e53216a1d2fc110f3c7c7dfb37135bec